modules/andes-core/broker/src/main/java/org/dna/mqtt/moquette/server/netty/NettyAcceptor.java

```diff
@@ -358,4 +358,49 @@ private SslHandler initSSLHandler(Properties props) {
         }
     }
 
+
+    private SslHandlerFactory initSSLHandlerFactory(Properties props) {
+        SslHandlerFactory factory = new SslHandlerFactory(props);
+        return factory.canCreate() ? factory : null;
+    }
+
+
+    private class SslHandlerFactory {
```

Comment:
Any possibility of defining this as a separate class ? instead of using inner classes